### ISSUE TITLE
I wanted to try and add a replace " x " (letter x U+0078) with " × " (multiplication sign, U+00D7)

### ISSUE THREAD

**Issue by mat813** [N-Parsons/grav-plugin-markdown-typography#1]
But I could not figure out how to plug into the markdown thingies, and all my tries were failures. :-(
I wonder if you could add this replacement to the plugin.

**Comment by N-Parsons** [comment]
It took me ages to work out and get used to the way these Markdown extensions are created, but they're pretty easy once you get the hang of it.

Anyway, the multiplication symbol is now available in v1.0.2 :)

**Comment by mat813** [comment]
Hi,

So, I today, I added a page for my Grav site, and it basically contains :

```markdown
---
author: Mathieu Arnold
title: Je suède, tu suèdes, il suède…
taxonomy:
    tag: [hahaha]
    category: blog
date: 2008-04-16 23:04:20 Z
updated: 2008-04-17 21:50:46.706954 Z
---

Récemment, je vous ai parlé de [Soyez sympas, rembobinez !][]{.flim}. Où suite
a un malencontreux accident, les personnages du film sont obligés de retourner
les films présents dans un vidéo club. Je suis tombé sur un suédage de
[King Kong][]{.flim} qui m’a beaucoup plu, alors, je vous [mets un lien][].

[Soyez sympas, rembobinez !]: ../2008-03-23-soyez-sympas-rembobinez
[King Kong]: http://french.imdb.com/title/tt0024216/
[mets un lien]: http://www.koreus.com/video/king-kong-suede
```

Fun fact, `malencontreux` matches the bits you added to support the multiplication sign, and breaks everything. I don't really understand why, it seems the whole text is in there and the brackets break the regexp engine because they're not escaped.
